docs/book/src/cronjob-tutorial/testdata/project/config/rbac/kustomization.yaml

```diff
@@ -7,11 +7,15 @@ resources:
 - service_account.yaml
 - role.yaml
 - role_binding.yaml
+- leader_election_role.yaml
+- leader_election_role_binding.yaml
+# The following RBAC configurations are used to protect
+# the metrics endpoint with authn/authz. These configurations 
+# ensure that only authorized users and service accounts
+# can access the metrics endpoint.
 - metrics_auth_role.yaml
 - metrics_auth_role_binding.yaml
 - metrics_reader_role.yaml
-- leader_election_role.yaml
-- leader_election_role_binding.yaml
```

Comment:
IHMO those needs be above so we can add the comments and make more clear what are the files used so users have a better chance to disable if they wish to do so.